hw11/mailroom.py

```diff
@@ -0,0 +1,96 @@
+
+
+def mailroom(userinput='thank you'):
+    fulldonorlist = [["Sydney Decoto", "Big Bird", "John Doe", "Jane Doe",
```

Comment:
The structure of this donor database seems a little confusing to me. I think it would be easier if your donors and their donations were grouped together.